crates/ruff_db/src/system/os.rs

```diff
@@ -16,12 +16,12 @@ use super::walk_directory::{
 };
 
 /// A system implementation that uses the OS file system.
-#[derive(Default, Debug)]
+#[derive(Default, Debug, Clone)]
 pub struct OsSystem {
     inner: Arc<OsSystemInner>,
 }
 
-#[derive(Default, Debug)]
+#[derive(Default, Debug, Clone)]
```

Comment:
This shouldn't require clone

```suggestion
#[derive(Default, Debug)]
```